index.html

```diff
@@ -202,11 +202,13 @@ <h2 class="title_card">About Myself</h2>
                             <img class="arrow_icon arrow_icon_active" src="images/icon_arrow.png" alt="arrow">
     
                         </li>
-                        <ul class="languages_content">
-                            <li><img src="images/js_icon.png" alt="javascript"><b class="language">JavaScript</b></li>
-                            <li><img src="images/html_icon.png" alt="html"><b class="language">HTML</b></li>
-                            <li><img src="images/css_icon.png" alt="css"><b class="language">CSS</b></li>
-                        </ul>
+                        <li>
+                            <ul class="languages_content">
+                                <li><img src="images/js_icon.png" alt="javascript"><b class="language">JavaScript</b></li>
+                                <li><img src="images/html_icon.png" alt="html"><b class="language">HTML</b></li>
+                                <li><img src="images/css_icon.png" alt="css"><b class="language">CSS</b></li>
+                            </ul>
+                        </li>
```

Comment:
- [ ] Kinldy make sure you fix the contrast of your colors for a better user experience on the button state (`hover`, `focus`, `active`) as you may notice how they are failing here.

`Hint: consider adjusting the color of the elements so that they may be really visible`
![image](https://user-images.githubusercontent.com/97100091/229848809-b3119a25-3c5d-4034-85ce-1182111d1249.png)
![image](https://user-images.githubusercontent.com/97100091/229849558-7ee98951-37e6-408a-bfee-5502f4dc3804.png)

Reply:
Thank you @christianbiring1. I changed the color of the button when hovering.